src/components/nav-logo/nav-logo.tsx

```diff
@@ -0,0 +1,104 @@
+import { Component, Element, EventEmitter, h, Host, Prop, Event } from "@stencil/core";
+import { CSS } from "./resources";
+
+@Component({
+  tag: "calcite-nav-logo",
+  styleUrl: "nav-logo.scss",
+  shadow: {
+    delegatesFocus: true
+  }
+})
+export class CalciteNavLogo {
+  //--------------------------------------------------------------------------
+  //
+  //  Element
+  //
+  //--------------------------------------------------------------------------
+  @Element() el: HTMLCalciteNavLogoElement;
+
+  //--------------------------------------------------------------------------
+  //
+  //  Public Properties
+  //
+  //--------------------------------------------------------------------------
+  /** When true, the component is highlighted. */
+  @Prop({ reflect: true }) active: boolean;
+
+  /** Specifies the URL destination of the component, which can be set as an absolute or relative path.*/
+  @Prop({ reflect: true }) href: string;
+
+  /** Describes the appearance or function of the `thumbnail`. If no label is provided, context will not be provided to assistive technologies. */
+  @Prop() label: string;
+
+  /** Specifies the subtext to display, such as an organization or application description. */
+  @Prop() subText: string;
+
+  /** Specifies the text to display, such as a product name.*/
+  @Prop() text: string;
+
+  /** When `true`, displays the `text` and `subText` contents. */
+  @Prop({ reflect: true }) textEnabled: boolean;
+
+  /** Specifies the `src` to an image. */
+  @Prop() thumbnail: string;
+
+  //--------------------------------------------------------------------------
+  //
+  //  Events
+  //
+  //--------------------------------------------------------------------------
+
+  /** Emits when the component's selection changes. */
+  @Event() calciteNavLogoSelect: EventEmitter<void>;
```

Comment:
Since there is no selection state, how is this (and other select events for navigation and menu components) different from a "click" event? If there's no distinction, I'd suggest removing them and have developers rely on the event target or provide specific event handlers to these components. 

For background: we did something similar some time ago with `action` and removed the redundant custom click event.